### ISSUE TITLE
Overlapping lines and text in AnimationPlayer editor Call Method track

### ISSUE THREAD

**Issue by fire-forge** [godotengine/godot#62984]
### Godot version

v4.0.alpha.custom_build [a6258fd79]

### System information

Windows 10

### Issue description

In the AnimationPlayer Call Method track, text can overlap and lines are drawn between keys that have the same function. Text should be clipped if it overlaps another key, and lines should not be drawn between the keys because they overlap the text.

![image](https://user-images.githubusercontent.com/67974470/178797504-e0732fb4-dc7c-4f1e-8477-8b799338c835.png)

In 3.5 the text overlapping issue exists but lines are not drawn between keys.

![image](https://user-images.githubusercontent.com/67974470/178799692-1c722013-49c1-42de-a2e9-5cf42926d621.png)

### Steps to reproduce

* Create a node
* Create an AnimationPlayer
* Open the Animation bottom tab
* Press "Add Track", select "Call Method Track", and select a node
* Insert multiple keys on the track using the same method
* Move the keys close enough that the text is overlapping

**Comment by KoBeWi** [comment]
Related: #27244

**Comment by Calinou** [comment]
The text ellipsis features added in 4.0 are likely helpful to avoid this.

We coul dalso use different text colors for odd/even keyframes.

**Comment by and-rad** [comment]
Would it also be an option to simply not display any text on the method track? No other track displays key values right there if I'm not mistaken, so why not be consistent about it? Clicking on the key shows the method name and arguments in the inspector, where they can be changed just like how it is for every other animation track.

**Comment by fire-forge** [comment]
@and-rad No, I like having the method names there. It can be useful to see what methods are in the animation at a glance. The text just shouldn't be overlapping.

**Comment by and-rad** [comment]
Fair enough, there's certainly value in being able to see which method is called at which point during the animation at a glance.

**Comment by aobrem** [comment]
Fixed by #96421 and #81563
